docs/healthcheck.md

```diff
@@ -0,0 +1,171 @@
+# Backbeat Healthcheck
+
+Backbeat exposes a healthcheck and a deep healthcheck route which return a
+response with an HTTP code.
+
+## Response Codes
+
+```
++----------+------------------------------------------------------------------+
+| Response | Details                                                          |
++==========+==================================================================+
+|   200    | OK: success                                                      |
++----------+------------------------------------------------------------------+
+|   403    | AccessDenied: request IP address must be defined in              |
+|          |   'conf/config.json' in field 'server.healthChecks.allowFrom'    |
++----------+------------------------------------------------------------------+
+|   404    | RouteNotFound: route must be valid                               |
++----------+------------------------------------------------------------------+
+|   405    | MethodNotAllowed: the HTTP verb must be a GET                    |
++----------+------------------------------------------------------------------+
+|   500    | InternalError: this could be caused by one of several            |
+|          |   components: the api server, Kafka, Zookeeper, or one of the    |
+|          |   Producers for a topic                                          |
++----------+------------------------------------------------------------------+
+```
+
+## Routes
+
+### `/_/healthcheck`
+
+Basic healthchecks return details on the health of Kafka and its topics.
+
+If no HTTP response error, the structure of the response is an array of 3
+key:value objects.
+
+One of these key:value objects will have keys of numbers, and values with an
+object of keys with `nodeId`, `host`, and `port`. The number keys represent
+each zookeeper node, and their respective details.
+
+```
+zookeeperNode: {
```

Comment:
I think it could be more clear to put this in angle braces to follow along with the text better. 
E.g., `<zookeeper node>`.